### PR TITLE
Add animation for loading more profile cards

### PR DESCRIPTION
**Pull request by Piumal1999**
## Purpose

To add an animation to load profile cards when the "See more" button on the team page is clicked.
Issue #279 

## Goals
Added a fade in animation for the cards

## Approach
JQuery and AJAX were used to add the animation.

### Screenshots
N/A
  
### Preview Link
https://piumal1999.github.io/sef-site/src/main/webapp/examples/team.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Related PRs
N/A

## Test environment
 
JDK 1.8, Ubuntu 19.04, Google Chrome

## Learning
Used Stackoverflow and W3Schools pages

### CODE REVIEW

File: src/main/webapp/examples/team.html [hunk 1]
```diff
@@ -121,8 +121,9 @@ <h5 class="title">
                     //hide button
                     $("#btnShowMore").click(function () {
                         let contentPartTwo = Mustache.render($("#templateTeam").html(), {"data": partTwo});
-                        $("#teamContent").append(contentPartTwo);
-                        document.getElementById("btnShowMore").style.display = "none";
+                        $(contentPartTwo).appendTo("#teamContent").hide().fadeIn(1000);
+
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/team.html]: Remove Whitespaces

File: src/main/webapp/examples/team.html [hunk 2]
```diff
@@ -132,7 +133,7 @@ <h5 class="title">
                     $("#teamContent").html(content);
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/team.html:133]: Remove whitespaces

**Review by jayasanka-sack — CHANGES_REQUESTED** [review]

File: src/main/webapp/examples/team.html (hunk 1 above)
Comment (Piumal1999) [reply src/main/webapp/examples/team.html]: done

File: src/main/webapp/examples/team.html (hunk 2 above)
Comment (Piumal1999) [reply src/main/webapp/examples/team.html:133]: done

**Comment by jayasanka-sack** [comment]
Merged.

**Pull request merged**